File: server/php/src/Application/Domain/Payments/PaymentDetails/AcquirerInformation.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails;

final class AcquirerInformation
{
    public ?AcquirerSelectionInformation $acquirerSelectionInformation = null;
    public ?string $name = null;

    public function toArray(): array
    {
        return [
            'acquirerSelectionInformation' => (
                $this->acquirerSelectionInformation ?? new AcquirerSelectionInformation()
            )->toArray(),
            'name' => $this->name,
        ];
    }
}

File: server/python/app/application/domain/payments/payment_details/acquirer_information.py
from dataclasses import dataclass
from typing import Optional
from app.application.domain.payments.payment_details.acquirer_selection_information import AcquirerSelectionInformation

@dataclass
class AcquirerInformation:
    acquirer_selection_information: Optional[AcquirerSelectionInformation] = None
    name: Optional[str] = None